flake.nix

```diff
@@ -6,9 +6,9 @@ rec {
 
   # These inputs are used for the manuals and release artifacts
   inputs.released-nixpkgs-unstable = { url = "nixpkgs/nixos-unstable"; };
-  inputs.released-nixpkgs-stable = { url = "nixpkgs/nixos-20.09"; };
+  inputs.released-nixpkgs-stable = { url = "nixpkgs/nixos-21.05"; };
   inputs.released-nix-unstable = { url = "github:nixos/nix/master"; };
-  inputs.released-nix-stable = { url = "github:nixos/nix/latest-release"; flake = false; };
+  inputs.released-nix-stable = { url = "github:jonringer/nix?ref=fix-2.3-build-stable"; flake = false; };
```

Comment:
shouldn't this be an official release instead of `github:jonringer`?

Reply:
it should, https://github.com/NixOS/nix/pull/4863 wasn't merged when I created the PR

Reply:
This was fixed